.rspec

```diff
@@ -0,0 +1,2 @@
+--color
+--require spec_helper
```

Comment:
You mean version controlling the .rspec file? I used to be against it, as I believe every developer on the project should be allowed to specify how their specs look and run. The only thing that's kinda annoying is that RSpec now seem to require having `--require spec_helper` in the .rspec file, which is something that essentially should be version controlled. If there's a way to avoid that, then I'd be against version controlling this file. :smiley: